ecosystem/sep-0012.md

```diff
@@ -16,36 +16,39 @@ This SEP defines a standard way for stellar clients to upload KYC (or other) inf
 
 This SEP was made with these goals in mind:
 
-* interoperability
-* Allow a customer to enter their KYC information to their wallet once and use it across many services without re-entering information manually
-* handle the most common 80% of use cases
-* handle image and binary data
-* support the set of fields defined in [SEP-9](sep-0009.md)
-* support authentication via [SEP-10](sep-0010.md)
-* support the provision of data for [SEP-6](sep-0006.md), [SEP-24](sep-0024.md), [SEP-31](sep-0031.md), and others
-* give customers control over their data by supporting complete data erasure
+- interoperability
```

Comment:
❗This is an active SEP. It's helpful if we keep changes surgical, so it's ultra clear to implementers what has changed. We can refactor but like code it's helpful if the refactor commits make no functional change so implementers can ignore them. can you move the format changes to a separate PR? I'd also question if the format changes are necessary. Even with refactors we should weigh the return on investment vs churning diffs and making it harder to follow the diffs history.

Reply:
Sorry, the reformatting happened automatically. just pushed it with the formatting removed.